### PR TITLE
Don't copy snowboy into susi_linux folder

### LINKED ISSUE #230: Dont  copy snowboy to susi_linux folder
**Is your feature request related to a problem? Please describe.**

This is in relation to https://github.com/fossasia/susi_linux/issues/219#issuecomment-402227526
**Describe the solution you'd like**

Remove operations where we are copying snowboy to susi_linux
**Describe alternatives you've considered**
NA
**Additional context**
NA

### PR DESCRIPTION
**Pull request by prateekiiest**
Fixes #230 

Also in relation to #219

### CODE REVIEW

**Comment by prateekiiest** [comment]
@hongquan reviews

@stealthanthrax please also have a look.

**Comment by prateekiiest** [comment]
@hongquan updated it

File: install.sh [hunk 1]
```diff
@@ -101,13 +101,9 @@ function install_snowboy()
     if install_dependencies
     then
         root_dir=$(pwd)
-        git clone https://github.com/Kitt-AI/snowboy.git
-        cd snowboy/swig/Python3
-        make -j4
+        sudo pip3 install git+https://github.com/Kitt-AI/snowboy.git
         if [ -f _snowboydetect.so ]; then
```
Comment (hongquan) [inline install.sh]: Snowboy now is no longer built inside susi_linux folder, this `if` line will fail.

Comment (prateekiiest) [reply install.sh]: I am not deleting the snowboy folder here @hongquan.
Will do that in the other PR

Comment (prateekiiest) [reply install.sh]: updated it

**Comment by prateekiiest** [comment]
updated it @hongquan . I am checking if the last exit status after installation was successful

**Comment by stealthanthrax** [comment]
@prateekiiest @hongquan , this PR has broken `config_generator.py ` and `main`scripts. We should move the contents of this PR to a single PR with this [PR](https://github.com/fossasia/susi_linux/pull/228) as this is putting the repo in a half working state.

**Pull request merged**